include/lsst/afw/math/BoundedField.h

```diff
@@ -37,6 +37,9 @@ namespace lsst { namespace afw { namespace math {
 /**
  *  @brief An abstract base class for 2-d functions defined on an integer bounding boxes
  *
+ *  Integer bounding boxes (afw.geom.Box2I) are inclusive of the end pixels, thus a BoundedField
+ *  is defined on the inclusive box [x0, x1] x [y0, y1].
```

Comment:
I think you need something stronger than this if you really want to get at what's not intuitive about these boxes.  Maybe something like this "because integer positions correspond to the centers of pixels, and these boxes include the entirety of those pixels, the actual range covered by the `BoundedField` is `[x0 - 0.5, x1 + 0.5] x [y0 - 0.5, y1 + 0.5]`."